app/javascript/components/admin/UserModal.js

```diff
@@ -109,18 +122,62 @@ class UserModal extends React.Component {
               />
             </Form.Group>
             {this.props.type === 'edit' && (
+              <Form.Row className="pt-4">
+                <Form.Group as={Col} md={8}>
+                  <Form.Label>System Access</Form.Label>
+                  <Form.Check
+                    id="system-access-input"
+                    name="system-access"
+                    type="switch"
+                    checked={this.state.isLocked}
+                    label={this.state.isLocked ? 'Locked' : 'Unlocked'}
+                    onChange={this.handleLockedSystemAccessChange}
+                  />
+                </Form.Group>
+                <Form.Group as={Col}>
+                  {this.state.lockReason === 'Auto-locked by the System' && this.state.isLocked && (
+                    <Form.Text as={Alert} id="system-access-auto-lock-reason" variant="light" className="admin-lock-alert">
+                      <i className="fas fa-exclamation-circle"></i>
+                      <span> {this.state.autoLockReason}</span>
+                    </Form.Text>
+                  )}
+                </Form.Group>
+              </Form.Row>
+            )}
+            {this.props.type === 'edit' && this.state.isLocked && (
               <Form.Group>
-                <Form.Label>Status</Form.Label>
-                <Form.Check
-                  id="status-input"
+                <Form.Label htmlFor="status-select">Status</Form.Label>
+                <Select
+                  inputId="status-select"
+                  id="status"
                   name="status"
-                  type="switch"
-                  checked={this.state.isLocked}
-                  label={this.state.isLocked ? 'Locked' : 'Unlocked'}
-                  onChange={this.handleLockedStatusChange}
+                  defaultValue={
```

Comment:
Can we use `value` here instead of `defaultValue`?  We don't typically use `defaultValue` (although this component would say otherwise), so it might be worth changing each of them for consistency

Reply:
Yes! See [be724b2f805fa4843309cff5302a8a597faefd26](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/be724b2f805fa4843309cff5302a8a597faefd26)

Reply:
email, jurisdiction, and role now used props to set the default value, so that will also need to be update to use state because they no longer update onChange

Reply:
I'm not sure if this addresses your comment, so let me know if I'm missing something. [64e730ff38c29993141bce68ac23b0f245a103e8](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/64e730ff38c29993141bce68ac23b0f245a103e8)

Reply:
yup behaves well now! had a couple cleanup comments though